devops/Makefile

```diff
@@ -14,6 +14,7 @@ export NEW_DB := $(or $(NEW_DB), $$NEW_DB)
 export JOB_NAME := $(or $(JOB_NAME), $$JN)
 export SITE_MINDER_LOGOUT_URL := $(or $(SITE_MINDER_LOGOUT_URL), https://logontest7.gov.bc.ca/clp-cgi/logoff.cgi)
 export PATRONI_SIMSDB_BACKUP_APP_NAME := $(or $(PATRONI_SIMSDB_BACKUP_APP_NAME), patroni-simsdb-backup)
+export PATRONI_SIMSDB_BACKUP_TAG_NAME := $(or $(PATRONI_SIMSDB_BACKUP_TAG_NAME), 2.7.0)
```

Comment:
What's "2.7.0". Shouldn't it be a constant? I know it is only used here but it would easier to understand what's this about.  Maybe a comment would be ok.

Reply:
Looks like 2.7.0 is the tag for the backup container version: https://github.com/BCDevOps/backup-container/tags but good question, a comment would be a good idea as I didn't understand at first either.

Reply:
The reason is as @cditcher mentioned, added a comment above to explain it. Also the assignment is a default value and its similar to mocking the package.json versioning, which represents that the tags minimum version used for proper working of the container.